Add additional profile for smaller binaries (#4)
cloudflare/app-relay-client-library: README.md

-Djava.library.path="lib/"
```

## Building size optimized binaries

> To build binaries with smaller disk footprint you can activate `release-space-optimized` profile:

Reviewer: ```suggestion
To build binaries with a smaller disk footprint you can use the `release-space-optimized` profile:
```

```sh

> # for ios

Reviewer: ```suggestion
# for iOS
```

➜  cargo build \
    --target aarch64-apple-ios \
    --no-default-features \
    --profile release-space-optimized

> # for android this will fail using ndk but the binaries will be located 

Reviewer: ```suggestion
# for Android this will fail using ndk but the binaries will be located 
```

> # in different directory the bug https://github.com/bbqsrc/cargo-ndk/issues/73

Reviewer: ```suggestion
# in different directory. See https://github.com/bbqsrc/cargo-ndk/issues/73
```

➜  cargo ndk \
    -t armeabi-v7a \
    -t arm64-v8a \
    -o ./target/jniLibs build --profile release-space-optimized
```

> For more background about the parameters set for this profile read [this repo](https://github.com/johnthagen/min-sized-rust)

Reviewer: ```suggestion
For more background about the parameters set for this profile read [this repo](https://github.com/johnthagen/min-sized-rust).
```
